skyportal/plot.py

```diff
@@ -1409,6 +1412,12 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, device="browser"):
         normfac = np.nanmedian(np.abs(s.fluxes))
         normfac = normfac if normfac != 0.0 else 1e-20
         altdata = json.dumps(s.altdata) if s.altdata is not None else ""
+        annotations = (
+            json.dumps([{a.origin: a.data} for a in s.annotations])
```

Comment:
Why is this necessary?

Reply:
Actually I'm not sure how the data would be rendered without this. I can give it a try.

Reply:
I think the json dump is needed to turn the objects into strings that bokeh can render.

Reply:
Okay